doc/source/how-to/releasing.rst

```diff
@@ -77,340 +83,442 @@ features without sacrificing stability. The model closely follows the
+    .. code-block:: text
+    
+       git add pyproject.toml
+       git commit -m "Bump version X.Y.0"
+    
+    Tag the previous commit using this command:
+    
+    .. code-block:: text
+    
+       git tag vX.Y.0
+    
+    Push the commit and the tag with these commands:
+    
+    .. code-block:: text
+    
+       git push -u origin release/X.Y
+       git push origin vX.Y.0
+
+
+.. dropdown:: Releasing patched versions
+
+    Patched versions allow you to fix issues discovered in published releases by
+    cherry-picking these fixes from the ``main`` branch.
+
+    Before performing a patch release, you must first identify which
+    ``release/X.Y`` branch it belongs to.
+    
+    .. code-block:: text
+    
+       git checkout release/X.Y
+       git fetch origin release/X.Y
+       git reset --hard origin/release/X.Y
+    
+    Now, use the following code to `cherry-pick <https://git-scm.com/docs/git-cherry-pick>`_
+    the fix commit from ``main``, which solves for the bug. Do not merge changes from
+    ``main`` into the release branch. Always cherry-pick them.
+    
+    .. code-block:: text
+       
+       git cherry-pick <commit hash>
+    
+    Ensure that your style, tests, and documentation checks are also passing.
+    
+    Increase by one unit the value of ``Z`` in your project version. Stash and
+    amend these new changes using this command:
+    
+    .. code-block:: text
+    
+       git add pyproject.toml
+       git commit --amend -m "Bump version X.Y.Z"
+    
+    Tag the previous commit with this command:
+    
+    .. code-block:: text
+    
+       git tag vX.Y.Z
+    
+    Push the commit and the tag using this command:
+    
+    .. code-block:: text
+    
+       git push -u origin release/X.Y
+       git push origin vX.Y.Z
+
+
+Publishing artifacts
+--------------------
+When a new version is released, some artifacts are provided with it. In Python,
+these :ref:`Artifacts` are typically the ``Wheel`` and ``Source`` files.
+Documentation in the form of PDF and HTML files are also considered artifacts.
 
-   git add . && git commit --amend -m "Bump version X.Y.Z"
+.. attention:: 
 
-Tag the previous commit with:
+   Do not distribute artifacts without approval. 
 
-.. code-block:: text
+   A project needs to be authorized to be released into public by following the
+   process explained in the :ref:`Project approval and public release` section.
 
-   git tag vX.Y.Z
 
-Push the commit and the tag with:
+There are three possible places where artifacts can be published:
 
-.. code-block:: text
+.. grid:: 3
+    
+    .. grid-item-card:: :octicon:`lock` Private PyPI
+       :link: managerial
+       :link-type: ref
 
-   git push -u origin release/X.Y && git push origin vX.Y.Z
+        This is a private index used to share artifacts across the company
+        while making sure that projects remain private.
 
+    .. grid-item-card:: :octicon:`unlock` Public PyPI
+       :link: legal
+       :link-type: ref
```

Comment:
Watch out...
* Card links are not pointing to the correct hyperlinks

Reply:
Good catch! Fixing this.